src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -64,16 +64,22 @@
   #include "stm32f4xx.h"
   #define EP_MAX          USB_OTG_FS_MAX_IN_ENDPOINTS
   #define EP_FIFO_SIZE    USB_OTG_FS_TOTAL_FIFO_SIZE
+  #define EP_MAX_HS       USB_OTG_HS_MAX_IN_ENDPOINTS
```

Comment:
I like the idea to include both FS and HS port endpoint number and FIFO here. Though we should make it consistent by changing EP_MAX to `EP_MAX_FS`, and maybe still keep the EP_MAX and EP_FIFO_SIZE naming as the selected one.

Reply:
Done